### ISSUE TITLE
Feature Request: Configuration File

### ISSUE THREAD

**Issue by kenderipa** [MicheleCioccarelli/blob-dl#6]
Hello, could you also add a configuration file (maybe based on the questionnaire)? Typing download path every time is tedious enough, but it subtitles get implemented someday, it'll get even worse. Thanks!

**Comment by MicheleCioccarelli** [comment]
Hey! Thank you for taking the time to make a feature request :)

I also think that using blob-dl with the same settings many times over can be tedious which is why I put in the option to view what command it generates (-s flag) so that you can copy that command and use it directly while just changing the url, but I see how that's hidden and not elegant

I think it would be a good idea to support config files and subtitles so I'll work on those.
I've been doing some bug fixes so I won't make a new release until I have everything ready, but I'll notify you when the newest build fully supports subtitles/config files.

If you have other suggestions about new features or user experience let me know

**Comment by MicheleCioccarelli** [comment]
In the latest release configuration files are supported and subtitles are downloaded by default.

**Comment by kenderipa** [comment]
Thanks! Another question: are the Linux binaries coming? There aren't any as of now.

**Comment by MicheleCioccarelli** [comment]
yeah its because the library I used for cross-compilation doesn't want to work with linux. Now I'm looking into it but it might take a few days to fix.

If you want to use the program right now you can just clone the repo, run "cargo build --release" and get your executable in the "target/release" directory

**Comment by kenderipa** [comment]
Thank, I'll wait, it's not urgent :)

**Comment by kenderipa** [comment]
So, um, still no Linux binaries :)

**Comment by kenderipa** [comment]
@MicheleCioccarelli, please see above.

**Comment by MicheleCioccarelli** [comment]
Sorry for the wait. 
In the newest release the linux binaries are available.

**Comment by kenderipa** [comment]
> Sorry for the wait. In the newest release the linux binaries are available.

Thank you!
